ansible/cleanup.yml

```
@@ -0,0 +1,74 @@
+---
+- hosts: production
+  become: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)